beacon-chain/powchain/log_processing.go

```diff
@@ -120,6 +149,44 @@ func (w *Web3Service) ProcessDepositLog(depositLog gethTypes.Log) {
 	}
 }
 
+// ProcessChainStart processes the log which had been received from
+// the ETH1.0 chain by trying to determine when to start the beacon chain.
+func (w *Web3Service) ProcessChainStart(genesisTime uint64) {
+	chainStartCount.Inc()
+
+	// w.chainStartETH1Data = &pb.Eth1Data{
```

Comment:
what is this?

Reply:
removed. it is being initialized in genesis state